_install-and-configure/upgrade-opensearch/index.md

```diff
@@ -91,8 +91,11 @@ Upgrades spanning more than a single major version of OpenSearch will require ad
 
 A rolling upgrade is a great option if you want to keep your cluster operational throughout the process. Data may continue to be ingested, analyzed, and queried as nodes are individually stopped, upgraded, and restarted. A variation of the rolling upgrade referred to as "node replacement" follows exactly the same process except that hosts and containers are not reused for the new node. You might perform node replacement if you are upgrading the underlying host(s) as well.
 
+[Segment replication]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/segment-replication/index/) could help you continue to index data on your primary shard as you being a rolling upgrade especially if your cluster has a high index count and lower replica shard count. Enabling segment replication gives you the ability upgrade nodes without need to disable than reenable replica shards, allowing indexing to continue even as you perform a rolling upgrade. For instructions about how to enable segment replication, see [Segment replication configuration]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/segment-replication/index/#segment-replication-configuration)
```

Comment:
Not sure if this is accurate. Enable segment replication, anyway need a full-reindex for the existing indexes. So, not sure if we can suggest users to use SegRep if they want to perform a rolling upgrade. I think we just remove the limitation and make sure users can do rolling upgrade while they are using SegRep.

Reply:
+1